client/src/components/Group/views/GroupServices.tsx

```diff
@@ -99,6 +122,18 @@ class GroupServices extends React.Component<IGroupProps, IGroupStates> {
       </>
     );
   }
+
+  private checkLegacy = () => {
+    fetchContainers(this.props.group._id)
+      .then((containers: IContainer[]) => {
+        containers.forEach(c => {
+          if (Object.entries(c.Labels).filter(l => l[0] === "SERVICE_NAME").length === 0) {
```

Comment:
if it's a legacy docktor service behind ssi does it work ?

Reply:
yes SERVICE_NAME is a docktor v2 label, SSO labels are PROJECT_NAME and SERVICE_TYPE